Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteLayerTreeDrawingArea.h

```diff
@@ -190,6 +192,7 @@ class RemoteLayerTreeDrawingArea : public DrawingArea, public WebCore::GraphicsL
     bool m_deferredRenderingUpdateWhileWaitingForBackingStoreSwap { false };
 
     Ref<WorkQueue> m_commitQueue;
+    Ref<WorkQueue> protectedCommitQueue() const;
```

Comment:
ditto.

Reply:
But also, can we make `m_commitQueue` const to avoid requiring a protected function?